core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.html

```diff
@@ -90,7 +97,7 @@
             }">
             <iframe
                 #iframe
-                [src]="(es.clientHost ? es.iframeURL : '') | safeUrl"
+                [src]="es.iframeURL | safeUrl"
```

Comment:
I think `"(es.clientHost ? es.iframeURL : '') | safeUrl"` is needed to load vtl pages, why the change here?

Reply:
While editing a VTL page, the `iframeURL` is empty. Therefore, I removed the conditional.

Is there any case when is a VTL Page and we have a `iframeURL`? @KevinDavilaDotCMS